src/main/java/bhs/devilbotz/utils/Utils.java

```diff
@@ -0,0 +1,18 @@
+package bhs.devilbotz.utils;
+
+import edu.wpi.first.wpilibj.Timer;
+
+public class Utils {
+  public static boolean isTrueForX(boolean input, double x) {
+    Timer timer = new Timer();
+    if (input) {
+      System.out.println("Timer Started");
+      timer.start();
+    } else {
+      System.out.println("Timer reset");
+      timer.stop();
+      timer.reset();
+    }
+    return timer.get() >= x;
+  }
+}
```

Comment:
As discussed last night, we don't need to push this to the main branch until at least it is being used.